src/dashboards/teachers/details.jsonnet

```
@@ -398,3 +398,900 @@ dashboard.new(
+          video_query: teachers_common.queries.video_id,
+        },
+        timeField: '@timestamp',
+      },
+    ],
+    transformations: [
+      {
+        id: 'convertFieldType',
+        options: {
+          conversions: [
+            {
+              targetField: common.fields.subtitle_enabled,
+              destinationType: 'string',
+            },
+          ],
+        },
+      },
+    ],
+    options: {
+      legend: {
+        displayMode: 'hidden',
+      },
+    },
+  },
+  gridPos={ x: 0, y: 26, w: 4, h: 6 }
+).addPanel(
+  {
+    type: 'barchart',
+    title: 'Subtitle language distribution',
+    description: |||
+      The distribution of subtitle languages chosen by users when it is activated.
+    |||,
+    options: {
+      legend: {
+        displayMode: 'hidden',
+      },
+    },
+    fieldConfig: {
+      overrides: [
+        {
+          matcher: {
+            id: 'byName',
+            options: 'Count',
+          },
+          properties: [
+            {
+              id: 'displayName',
+              value: 'Occurencies',
+            },
+          ],
+        },
+      ],
+    },
+    targets: [
+      {
+        datasource: common.datasources.lrs,
+        query: '%(video_query)s AND verb.id:"interacted"' % {
+          video_query: teachers_common.queries.video_id,
+        },
+        metrics: [common.metrics.count],
+        bucketAggs: [
+          {
+            id: '2',
+            type: 'terms',
+            settings: {
+              min_doc_count: '1',
+              size: '10',
+              order: 'desc',
+              orderBy: '_term',
+            },
+            field: common.fields.subtitle_language,
+          },
+        ],
+        timeField: '@timestamp',
+      },
+    ],
+    datasource: common.datasources.lrs,
+  },
+  gridPos={ x: 4, y: 26, w: 4, h: 6 }
+)
+.addPanel(
+  {
+    type: 'barchart',
+    title: 'Full screen activation',
+    datasource: common.datasources.lrs,
+    description: |||
+      The distribution of full screen activation.
+      By default they are deactivated.
+    |||,
+    fieldConfig: {
+      overrides: [
+        {
+          matcher: {
+            id: 'byName',
+            options: common.fields.full_screen,
+          },
+          properties: [
+            {
+              id: 'mappings',
+              value: [
+                {
+                  type: 'value',
+                  options: {
+                    '0': {
+                      text: 'off',
+                      index: 0,
+                    },
+                    '1': {
+                      text: 'on',
+                      index: 1,
+                    },
+                  },
+                },
+              ],
+            },
+          ],
+        },
+        {
+          matcher: {
+            id: 'byName',
+            options: 'Count',
+          },
+          properties: [
+            {
+              id: 'displayName',
+              value: 'Occurencies',
```

Should be:
```suggestion
              value: 'Occurrences',
```
everywhere.